var/spack/repos/builtin/packages/touchdetector/package.py

```diff
@@ -40,15 +40,17 @@ class Touchdetector(CMakePackage):
 
     depends_on('cmake', type='build')
     depends_on('boost@1.50:')
+    depends_on('eigen', when='@4.5:')
```

Comment:
eigen has following `True` variants: `+fftw+metis+mpfr+scotch+suitesparse`. Most likely these are not needed for touch detector (same as NMODL). could you turn them off here?

Reply:
OK.

Reply:
> eigen has following `True` variants: `+fftw+metis+mpfr+scotch+suitesparse`. Most likely these are not needed for touch detector (same as NMODL). could you turn them off here?

Do you have this recipe somewhere private?